include/riak_kv_ts.hrl

```diff
@@ -29,12 +29,12 @@
 %% For dialyzer types
 -include_lib("riak_ql/include/riak_ql_ddl.hrl").
 
--define(SQL_SELECT, #riak_select_v1).
--define(SQL_SELECT_RECORD_NAME, riak_select_v1).
+-define(SQL_SELECT, #riak_select_v2).
+-define(SQL_SELECT_RECORD_NAME, riak_select_v2).
 
 %% the result type of a query, rows means to return all matching rows, aggregate
 %% returns one row calculated from the result set for the query.
--type select_result_type() :: rows | aggregate.
+-type select_result_type() :: rows | aggregate | group_by.
```

Comment:
This change was made knowing that we know that the result type is not checked by the other vnodes, in the future we may either need this information to execute on the vnode, or have to use capabilities to make sure we are sending truly compatible records.